src/js/components/about_section.js

```diff
@@ -1,18 +1,16 @@
 Vue.component('about-section', {
   template: `
-    <div class="about">
+    <section class="about">
       <h2 class="about-title">このアプリについて</h2>
 
       <div class="help-sections">
         <div class="help-section">
-          <div class="help-text">
-            &#x1F441;を押すと、入力の文字に似てる部首を検索出来ます。検索対象になった似た部首には&#x1F441;が付きます。
-          </div>
+          <p class="help-text"><i class="far fa-eye"></i>を押すと、入力の文字に似てる部首を検索出来ます。検索対象になった似た部首には&#x1F441;が付きます。</p>
```

Comment:
~~Ah, sorry, could you change the `&#x1F441;` to `<i class="far fa-eye"></i>` as well?~~

Oh wait, the actual icon will look different... Hmm... Okay, you can forget about that. I'll fiddle with the radical selection to use that instead.